packages/sources/vue/src/components/VtmnLink/VtmnLink.vue

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+import '@vtmn/css-link/dist/index.css';
+import { reactive, computed, defineComponent } from 'vue';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnLink',
+  props: {
+    size: {
+      type: String,
+      default: 'medium',
+      validator: (val: string) => ['small', 'medium', 'large'].includes(val),
+    },
+    isStandalone: {
+      type: Boolean,
+      default: false,
+    },
+    hasIconAlong: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  setup(props) {
+    props = reactive(props);
+
+    return {
+      classes: computed(() => ({
+        'vtmn-link': true,
+        [`vtmn-link_size--${props.size}`]: true,
+        'vtmn-link--standalone': props.isStandalone,
+        'vtmn-link--icon-along': props.hasIconAlong,
```

Comment:
I think the component needs more logic because if the link is inline it can't have an icon along.

Reply:
I take the same algorithm as Svelte, React and Web Components, so yes we will need to refactor this in all stack

Reply:
@GaspardMathon we still need to add the same algorithm to other frameworks.